src/frontend/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -126,11 +126,6 @@ export class VideoPlayer extends React.Component<
           crossOrigin="anonymous"
           poster={video.urls.thumbnails[720]}
         >
-          <source
-            src={video.urls.manifests.hls}
-            size="auto"
-            type="application/vnd.apple.mpegURL"
-          />
           {!this.state.isDashSupported &&
```

Comment:
This condition should be removed ? In case where dashboard is supported, there is no more `source` available?

Reply:
When Dash is supported we're using Dash so we don't need those sources. They only add UI elements that can not be used because Dash overrides them.